docs/source/analysis.rst

```diff
@@ -13,6 +13,7 @@ Functions
   compute_entropy_gain <_autosummary/dynsight.analysis.compute_entropy_gain>
   compute_shannon_multi <_autosummary/dynsight.analysis.compute_shannon_multi>
   compute_entropy_gain_multi <_autosummary/dynsight.analysis.compute_entropy_gain_multi>
+  sample_entropy <_autosummary/dynsight.analysis.sample_entropy>
```

Comment:
Keep public, but state in docs that people should use `compute` functions. Same for pairwise.